spec/models/movie_spec.rb

```diff
@@ -39,9 +39,15 @@
     end
 
     describe ".most_attended" do
+
       it "returns array of three movies with most group participants" do
         expect(Movie.most_attended(3)).to eq [gold_movie, silver_movie, bronze_movie]
       end
+
+      it "returns empty array if there are no movies in the database" do
+        Movie.delete_all
```

Comment:
Interesting caution, but it should not be necessary (otherwise, there is a worry I think).